lib/windows/winlean.nim

```diff
@@ -1129,5 +1132,39 @@ proc setFileTime*(hFile: Handle, lpCreationTime: LPFILETIME,
                  lpLastAccessTime: LPFILETIME, lpLastWriteTime: LPFILETIME): WINBOOL
      {.stdcall, dynlib: "kernel32", importc: "SetFileTime".}
 
+type
+  SID_IDENTIFIER_AUTHORITY* = object
+    value*: array[6, BYTE]
+  PSID_IDENTIFIER_AUTHORITY* = ptr SID_IDENTIFIER_AUTHORITY
+  SID* = object
```

Comment:
how stable are those declarations?
shouldn't that be
```nim
type SID* {.importc.} = object
  ... 
```
?
otherwise, if these declarations change, things may break.

Reply:
OK. I'll fix this tomorrow.

Reply:
Fixed